sky/clouds/gcp.py

```diff
@@ -28,7 +28,7 @@
 # gcloud:
 # https://cloud.google.com/docs/authentication/provide-credentials-adc#local-key
 _GCP_APPLICATION_CREDENTIAL_ENV = 'GOOGLE_APPLICATION_CREDENTIALS'
-DEFAULT_GCP_APPLICATION_CREDENTIAL_PATH: str = os.path.expanduser(
+DEFAULT_GCP_APPLICATION_CREDENTIAL_PATH: str = (
```

Comment:
How about something like 
```
# NOTE: do not expanduser() on this path. It's used as a destination path on the remote serve.
```
or something better. Similarly for L37/40/51 if applicable?